test/Constraints/function_builder_diags.swift

```diff
@@ -166,7 +166,7 @@ struct Label<L> : P where L : P { // expected-note {{'L' declared as parameter t
 }
 
 func test_51167632() -> some P {
-  AnyP(G { // expected-error {{static method 'buildBlock' requires that 'Label<Any>.Type' conform to 'P'}}
+  AnyP(G { // expected-error {{metatype type 'Label<Any>.Type' cannot conform to 'P'; only struct/enum/class types can conform to protocols}}
```

Comment:
Loosing a note here is a regression, I think it's always useful to point exactly where the requirement is specified and what the substitution was if possible...